### PR TITLE
scrollview: reanimate QUnit tests

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: testing/helpers/wrappers/tabsWrappers.js [hunk 1]
```diff
@@ -32,13 +32,13 @@ export class TestTabsWrapper {
...
     checkTabsWithoutScrollable() {
-        const $tabs = this._$tabs.find('> .dx-tabs-wrapper > .dx-tab');
+        const $tabs = this._$tabs.find('.dx-tabs-wrapper > .dx-tab');
         QUnit.assert.equal($tabs.length, this._tabs.option('items').length, 'tabs are rendered');
         this._checkTabsContent($tabs);
     }
 
     checkTabsWithScrollable() {
-        const $tabs = this._$tabs.find('> .dx-tabs-scrollable .dx-scrollable-content > .dx-tabs-wrapper > .dx-tab');
+        const $tabs = this._$tabs.find('> .dx-tabs-scrollable .dx-scrollable-content .dx-tabs-wrapper > .dx-tab');
```
Comment (IgnatovDan) [inline testing/helpers/wrappers/tabsWrappers.js]: Let's extract such changes to a new PR and review why we need them.

**Review by IgnatovDan — DISMISSED** [review]

File: testing/helpers/wrappers/tabsWrappers.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply testing/helpers/wrappers/tabsWrappers.js]: This changes need us while our 'stopper' problems not fixed. After that problems will fix I am going to revert such changes.

**Review by IgnatovDan — APPROVED** [review]

**Review by IgnatovDan — DISMISSED** [review]

File: js/ui/grid_core/ui.grid_core.rows.js [hunk 2]
```diff
@@ -923,7 +923,10 @@ export const rowsModule = {
 
                     if(dxScrollable) {
                         dxScrollable.update();
-                        this._updateHorizontalScrollPosition();
+                        // TODO: seems that it method doesn't need for renovated widget
```
Comment (IgnatovDan) [inline js/ui/grid_core/ui.grid_core.rows.js]: Can we add one new test for this 'doesn't need' and run it on both the current sources and renovated scrollview?

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: js/renovation/ui/scroll_view/scrollbar.tsx [hunk 3]
```diff
@@ -438,7 +402,7 @@ export class Scrollbar extends JSXComponent<ScrollbarPropsType>() {
   }
 
   get inRange(): boolean {
```
Comment (IgnatovDan) [inline js/renovation/ui/scroll_view/scrollbar.tsx]: It's difficult to understand how the 'inRange' works by its name:

- Component name is 'ScrollBar'
- Function name is 'inRange'
- Checked variable is scrollLocation
- Checked min/max variables are min/maxOffset

**Review by IgnatovDan — DISMISSED** [review]

**Pull request closed without merging**